client/package.json

```diff
@@ -9,11 +9,15 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@hookform/resolvers": "^3.3.4",
     "axios": "^1.6.8",
     "next": "^14.2.3",
     "next-themes": "^0.3.0",
+    "nextjs-toploader": "^1.6.12",
```

Comment:
Here also we don't need external package, You can take loaders for tailwind which is already available